examples/tpm2-ekcert/main.go

```diff
@@ -51,6 +47,22 @@ var (
 	}
 )
 
+// Auth Policy calculations in Credential_Profile_EK_V2.0, section 2.1.5.3, "authPolicy"
+func defaultAuthPolicy() []byte {
+	previousDigest := make([]byte, 32)            // All zero buffer
+	policySecretCommandCode := uint32(0x00000151) // TPM__CC_PolicySecret
+	name := uint32(0x4000000B)                    // TPM_RH_ENDORSEMENT
+	policyRef := make([]byte, 0)
+
+	buf := bytes.NewBuffer(previousDigest)
+	binary.Write(buf, binary.BigEndian, policySecretCommandCode)
+	binary.Write(buf, binary.BigEndian, name)
+	digest1 := sha256.Sum256(buf.Bytes())
+
+	digest2 := sha256.Sum256(append(digest1[:], policyRef...))
```

Comment:
policyRef is empty. This is identical to `digest2 := sha256.Sum256(digest1[:])`

Reply:
Fixed, this was intended to show how a non-trivial auth policy would be computed. However, a comment is better suited for this purpose.